frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx

```diff
@@ -176,8 +176,21 @@ export const taxonomicFilterLogic = kea<taxonomicFilterLogicType>({
                     type: TaxonomicFilterGroupType.EventProperties,
                     endpoint: combineUrl(
                         `api/projects/${teamId}/property_definitions`,
-                        featureFlags[FEATURE_FLAGS.UNSEEN_EVENT_PROPERTIES] ? { event_names: eventNames } : {}
+                        featureFlags[FEATURE_FLAGS.UNSEEN_EVENT_PROPERTIES] && eventNames.length > 0
+                            ? { event_names: eventNames }
+                            : {}
                     ).url,
+                    scopedEndpoint:
+                        featureFlags[FEATURE_FLAGS.UNSEEN_EVENT_PROPERTIES] && eventNames.length > 0
+                            ? combineUrl(`api/projects/${teamId}/property_definitions`, {
+                                  event_names: eventNames,
+                                  is_event_property: true,
+                              }).url
+                            : undefined,
+                    expandLabel: ({ count, expandedCount }) =>
+                        `Show ${expandedCount - count} ${
+                            expandedCount - count === 1 ? 'property' : 'properties'
+                        } that haven't been seen with this event`,
```

Comment:
If I have more than one series, I'm not sure if "this event" refers to one of the series, or both combined.
Also, what happens if just one of the series is an action?

Reply:
Fixed that by saying "this" or "these" as appropriate.

If one of the series is an action that contains an event in any of its steps, `eventNames` will include them here.